web/src/Web.App/Domain/ComparatorSet.cs

```diff
@@ -12,5 +12,5 @@ public record ComparatorSet
 
 public record ComparatorSetUserDefined
 {
-
+    public string[] Set { get; set; } = Array.Empty<string>();
```

Comment:
Is `Set` too generic a name? Are these only ever going to be URNs?

Reply:
Yeah, it's just a list of URN's

Reply:
Not keep on changing as it's mapped all the way to the db column